### PR TITLE
Fix/3230 Formatting Terms of Use (EN)

### PR DESCRIPTION
**Pull request by tcfos**
## Desription
This PR fixes some formatting issues in the EULA.

## Jira 
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-3230

## Screenshots

Terms of Use EN

### CODE REVIEW

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/usage.html [hunk 1]
```diff
@@ -174,45 +174,19 @@
...
+		<p>4. The App does not protect you against infection</p>
+		<p>5. Retrieval of test results</p>
+		<p>6. Activating the warning feature</p>
+		<p>7. Requirements for using the App</p>
+		<p>8. Limitations of the App</p>
+		<p>9. Rights of use</p>
+		<p>10. Important information about availability and changes</p>
+		<p>11. No warranty</p>
+		<p>12. Special terms for the iOS version of the App</p>
+		<p>13. Final provisions</p>
```
Comment (whiskey) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/usage.html:189]: We really need a sophisticated solution here. Not only will, for example an `<ol>`, create cleaner semantics but also help with accessibility for screen readers!

**Review by whiskey — APPROVED** [review]
Approved. Just added my 0.02€.

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/usage.html (hunk 1 above)
Comment (tcfos) [reply src/xcode/ENA/ENA/Resources/Localization/en.lproj/usage.html:189]: Converting the MSWord document to html is a pain. There is still manual formatting required, and the CSS styles  are not working reliably.

**Review by janetback — APPROVED** [review]
UA reviewed

**Pull request merged**